go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -872,7 +872,7 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 						nodeSubnets, err := util.ParseNodeHostSubnetAnnotation(&node, types.DefaultNetworkName)
 						if err != nil {
 							err1 = fmt.Errorf("unable to fetch node-subnet annotation for node %s: err, %v", node.Name, err)
-							return false, nil
+							continue
```

Comment:
hm... so, if it turns out that no nodes have a subnet annotation, we will skip them all and that is ok? In other words, what we really care here is that if the node has a subnet annotation, it better pass `checkOVNSBNodeLRSR`. Either that, or it is okay for it not to have that annotation. Right?

Reply:
right if the node is plumbed in, then we expect the route to exist for overlay traffic, if it doesn't have the node add done correctly because of lack of annotation, then we ignore that node